Cargo.toml

```diff
@@ -19,6 +19,12 @@ features = ["derive"]
 version = "0.7.6"
 features = ["critical-section-single-core"]
 
+[workspace.dependencies.mavlink]
```

Comment:
Not sure if this belongs here. Since both the main board and the ground station need to have this in sync, I'm thinking that only `messages` should have a dependency on mavlink. Then, `messages` can export this dependency by doing a `pub use mavlink` at the top level crate. Then everything else uses mavlink through `messages`. This would help keep everything in sync.